### PR TITLE
Updated evaluate to take actual calculated values from the calculatio…

### PR DESCRIPTION
**Pull request by MadsHeilmann**
…n function. Also updated the default input to be more interesting, such taht we get different routes.

### CODE REVIEW

**Comment by SebastianMygind** [comment]
> …n function. Also updated the default input to be more interesting, such taht we get different routes.

Frederikssund slander

File: src/calculations.h [hunk 1]
```diff
@@ -44,7 +44,7 @@ typedef struct CalculateCo2Parameters {
   const int kFuelEfficiency;
 } CalculateCo2Parameters;
 
-double CalculateDistance(
+int CalculateDistance(
```
Comment (TexWO) [inline src/calculations.h]: Why int?

**Comment by SebastianMygind** [comment]
When running the program and choosing preset at the first prompt, it fails with the error message (No stops: Success

Process finished with exit code 1).

It seems to be an expected error. But maybe change origin and destination?

File: src/calculations.h (hunk 1 above)
Comment (MadsHeilmann) [reply src/calculations.h]: Actually dont know. I have changed it back. My bad

**Comment by MadsHeilmann** [comment]
> When running the program and choosing preset at the first prompt, it fails with the error message (No stops: Success
> 
> Process finished with exit code 1).
> 
> It seems to be an expected error. But maybe change origin and destination?

I agree. I have raised the issue with Martin. It seems to be with the API - but i am not sure. I will nevertheless not merge before it is cleared.

**Review by SebastianMygind — DISMISSED** [review]

**Comment by MadsHeilmann** [comment]
In this PR version of the program it functions properly. Time, enviornment and price also calculates correctly.

**Review by SebastianMygind — DISMISSED** [review]

**Review by SebastianMygind — APPROVED** [review]
nice

**Review by Danar435 — APPROVED** [review]

**Pull request merged**